components/test_raftstore/src/util.rs

```diff
@@ -1897,3 +1897,13 @@ pub fn wait_down_peers<T: Simulator>(cluster: &Cluster<T>, count: u64, peer: Opt
         peers, count, peer
     );
 }
+
+pub fn get_region_read_index_safe_ts<T: Simulator>(
```

Comment:
If if cannot be `cfg(test)`, maybe we should leave a comment stating it's only for tests.